index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Random Word Pick</title>
+</head>
+<body>
+    <div class="adminCommandsWrapper">
+        <button id="submitModeButton" onclick="buttonPressed(0);" class="clearButton">Submit mode</button>
+        <button id="clearButton" onclick="buttonPressed(1);" class="clearButton">Clear</button>
+        <input type="password" onblur="clearWordsBarBlur();" placeholder="password" id="clearWordInput" class="clearWordBar">
+    </div>
+  
+    <h1>Random Word Picker</h1>
+  
+    <div class="buttonWrapper">
+        <button onclick="pickWord();" class="bigButton">Pick Word</button>
+        <h2 id="word"></h2>
+        <input type="text" class="addWord" placeholder="Add Word" id="addWordInput">
```

Comment:
I think you can just use onSubmit for your case

https://www.w3schools.com/tags/ev_onsubmit.asp